src/libraries/System.Threading.Tasks/tests/CancellationTokenTests.cs

```diff
@@ -379,6 +379,37 @@ public static void CancellationTokenWaitHandle_WaitAny()
                "CancellationTokenWaitHandle_WaitAny:  The token should have been canceled.");
         }
 
+        [Fact]
+        public static void CreateLinkedTokenSource_OneToken()
+        {
+            CancellationTokenSource original;
+
+            original = new CancellationTokenSource();
```

Comment:
what would be the behavior if we have a chain of linked token sources? I mean 


original = new CancellationTokenSource();
            using (var linked1 = CancellationTokenSource.CreateLinkedTokenSource(original))
            using (var linked2 = CancellationTokenSource.CreateLinkedTokenSource(linked1))
           {
                 original.Cancel();
           }

and is it worth it to test such case?

Reply:
We could, but I don't think it's necessary.  Linking just creates another CTS, so as long as one hop works, we should be able to assume that they work transitively (and we have tests for other cases of that elsewhere).